common-lib/src/main/scala/com/gu/mediaservice/lib/argo/model/EmbeddedEntity.scala

```diff
@@ -19,7 +20,8 @@ object EmbeddedEntity extends WriteHelpers {
   implicit def embeddedEntityWrites[T: Writes]: Writes[EmbeddedEntity[T]] = (
     (__ \ "uri").write[String].contramap((_: URI).toString) ~
       (__ \ "data").writeNullable[T] ~
-      (__ \ "links").writeNullable[List[Link]].contramap(someListOrNone[Link])
+      (__ \ "links").writeNullable[List[Link]].contramap(someListOrNone[Link]) ~
+      (__ \ "actions").writeNullable[List[Action]].contramap(someListOrNone[Action])
```

Comment:
A concern about this being none - with links - we know we are going to have a specific set of links.
With actions they seem more conditional, i.e. you can't delete, no delete action.

This leaves the API with two shapes:
```JSON
{
  "data": {},
  "actions": [ {"detete": {} }]
}
```
& 
```JSON
{
  "data": {}
}
```

On the client this would mean we would have have annoying client code of:
```JavaScript
if (actions ** getAction('delete')) {/*magical sparkles*/}
```

Although thinking about it - I am assuming this will be done in theseus - so it's a non-problem?